crates/primitives/chain-id/src/tests.rs

```diff
@@ -0,0 +1,27 @@
+use starknet_ff::FieldElement;
+
+use crate::*;
+
+#[test]
+fn test_sn_goerli_chain_id() {
+    let sn_goerli_chain_id = FieldElement::from_byte_slice_be(b"SN_GOERLI").unwrap();
+    let mont_sn_goerli_chain_id = FieldElement::into_mont(sn_goerli_chain_id);
+    let expected_value: Felt252Wrapper = Felt252Wrapper(starknet_ff::FieldElement::from_mont(mont_sn_goerli_chain_id)).into();
```

Comment:
Why are you doing this? It looks useless

Can't you do this instead:
```rust
let expected_value = Felt252Wrapper(FieldElement::from_byte_slice_be(b"SN_GOERLI").unwrap());
assert_eq!(
        SN_GOERLI_CHAIN_ID, 
        expected_value, 
        "SN_GOERLI_CHAIN_ID does not match the expected value."
    );
```

Reply:
Right, I forgot the from_mont in my declaration